kolibri/plugins/device/kolibri_plugin.py

```diff
@@ -16,15 +18,30 @@ class DeviceManagementPlugin(KolibriPluginBase):
     translated_view_urls = "urls"
 
 
+def any_ie11_users():
+    from kolibri.core.logger.models import UserSessionLog
+
+    return UserSessionLog.objects.filter(device_info__contains="IE,11").count() > 0
```

Comment:
Oh, one thing - you should change this to:
```suggestion
    return UserSessionLog.objects.filter(device_info__contains="IE,11").exists()
```

Count will force it to explicitly count all the rows in the DB to which this applies, whereas exists just checks that 1 exists and stops there. Should almost always be faster (but will never be slower).